VoteCounter/src/GUI.cs

```diff
@@ -27,48 +27,54 @@ private void btnGenerateVoteCount_Click(object sender, EventArgs e)
 
             if (Int32.TryParse(txtStartingPost.Text, out startingPostNumber) && Int32.TryParse(txtEndingPost.Text, out endingPostNumber))
             {
-                PostList postList = new PostList(url, startingPostNumber, endingPostNumber);
-
                 //make a List
                 List<Player> players = new List<Player>();
 
                 //set the players
                 foreach (DataGridViewRow row in listPlayers.Rows)
                 {
-                    List<string> nicks = new List<string>();
-                    string name = "";
-                    for (int i = 0; i < listPlayers.Columns.Count; i++)
+                    Player newPlayer = new Player("");
```

Comment:
And if I just add Player(){} to it it returns a nullexception when I try to add to it.